### DISCUSSION TITLE
Pi5 Help

### DISCUSSION THREAD

**Discussion by adingess** [DISTRHO/Cardinal#634] [Q&A]
Hi team,
   I love the software!
I have Ubuntu (Pi flavor) running on a Pi5 which is running Cardinal standalone and Reaper with Cardinal VST.

(2) Minor issues that I was hoping there is a fix for?
1.) I can't seem to use my touchscreen to right click to add new modules. Right clicking works in the OS by holding a finger down for 3 sec. This doesn't work within Cardinal, any suggestions to get right clicking working via touchscreen?
2.) Is there anyway to add Cardinal standalone to the apps menu or a way to pin it to the taskbar for quick launch?

Is Ubuntu on a Pi5 the way to go or would Raspbian be a better solution here?

Thanks for the help and keep up the great work! <3

**Comment by dromer** [comment]
1) Rack doesn't have any touch extensions, this may also need additions in pugl (the library used for user input).
2) This would be fixed by having https://github.com/DISTRHO/Cardinal/issues/632

I don't think there would be any significant difference between Ubuntu and Raspbian. Just make sure to use a very minimal and lightweight desktop environment/window manager.

**Reply by adingess** [reply]
Thank you so much for the info and your time.
Do you feel that these 2 issues would warrant a ticket or request to add in the se features?
My goal is to velcro a Pi5 behind my touchscreen monitor and have a really nice modular synth.

**Reply by dromer** [reply]
Both issues have a ticket already. Second one I linked already.
First one: https://github.com/DISTRHO/Cardinal/issues/527

**Reply by adingess** [reply]
Brilliant! LMK if there's a donations page and I'd be happy to chip in for these features <3
